src/features/commands.ts

```diff
@@ -24,35 +24,29 @@ export default function registerCommands(server: OmniSharpServer, platformInfo:
     let d1 = vscode.commands.registerCommand('o.restart', () => restartOmniSharp(server));
     let d2 = vscode.commands.registerCommand('o.pickProjectAndStart', async () => pickProjectAndStart(server, optionProvider));
     let d3 = vscode.commands.registerCommand('o.showOutput', () => eventStream.post(new ShowOmniSharpChannel()));
-    let d4 = vscode.commands.registerCommand('dotnet.restore', fileName => {
-        if (fileName) {
-            dotnetRestoreForProject(server, fileName, eventStream);
-        }
-        else {
-            dotnetRestoreAllProjects(server, eventStream);
-        }
-    }); 
+    let d4 = vscode.commands.registerCommand('dotnet.restore.project', () => pickProjectAndDotnetRestore(server, eventStream));
```

Comment:
This is getting a bit silly. Can we just start pushing these into an array and construct our composite disposable from that?